test/SlowTests/Cluster/ClusterOperationTests.cs

```diff
@@ -254,20 +248,21 @@ public async Task PreferredNodeShouldBeRestoredAfterBroadcast()
         [Fact]
         public async Task ChangesApiFailOver()
         {
-            using (var cts = new CancellationTokenSource(TimeSpan.FromMinutes(3)))
+            DebuggerAttachedTimeout.DisableLongTimespan = true;
```

Comment:
this should be removed?

Reply:
Yep. Done

Reply:
Oh, oops my mistake. 
The `DebuggerAttachedTimeout.DisableLongTimespan = true;` is fine as far as I know, and now we can remove the `CancellationTokenSource` (what I did in this fix). let me know if you want it back  :-)

Reply:
CTS was there to timeout the test if needed, I would not like this test to hang.

Reply:
OK sorry. Done (returned CTS and keeping `DebuggerAttachedTimeout.DisableLongTimespan = true;`)

Reply:
But this is static and affects other tests, isnt this for debugging purposes only?

Reply:
Done